modules/commons/src/main/resources/schemas/neurosciencegraph/commons/protocol/v0.1.5.json

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/commons/protocol/v0.1.4/shapes/"
```

Comment:
Update to the current commons/protocol/v0.1.5

Reply:
Done.